packages/serverless/src/model/database/group.ts

```diff
@@ -39,14 +40,20 @@ export class ModelGroup extends ModelGeneral<WithoutId<TGroupRecord>> {
     return this.collection.findOne({ groupName }, { session });
   }
 
-  public static async init(databaseName: string) {
+  public static async init(databaseName: string, session?: ClientSession) {
     const collection = ModelCollection.getInstance(
       databaseName,
       DB.service,
       ModelGroup.collectionName
     );
+    /*
+      groupName: string;
+      groupDescription: string;
+    */
     if (!(await collection.isExist())) {
-      await collection.index({ groupName: 1 }, { unique: true });
+      collection.index({ groupName: 1 }, { unique: false, session });
```

Comment:
We cannot set index index within transaction

Reply:
resolved. I add all `await`